Formula/opendht.rb

```diff
@@ -0,0 +1,39 @@
+class Opendht < Formula
+  desc "C++14 Distributed Hash Table implementation"
+  homepage "https://github.com/savoirfairelinux/opendht"
+  url "https://github.com/savoirfairelinux/opendht/archive/refs/tags/2.3.4.tar.gz"
+  sha256 "3c902f30bae21c6ac1ff08417f01c9a6ad252db32afac72508ed321d8478c5ab"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "argon2"
+  depends_on "asio"
+  depends_on "gnutls"
+  depends_on "msgpack-cxx"
+  depends_on "nettle"
+  depends_on "readline"
+
+  def install
+    system "cmake", ".", "-DOPENDHT_C=ON", "-DOPENDHT_TOOLS=ON", *std_cmake_args
+    system "make", "install"
```

Comment:
I think there has been a recent preference in Homebrew style to letting `cmake` manage the entire build/install, e.g.
```suggestion
    system "cmake", "-S", ".", "-B", "build", "-DOPENDHT_C=ON", "-DOPENDHT_TOOLS=ON", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

For example, if you use `brew create --cmake <URL>`, it will use this style

Reply:
Should brew audit flag this up then?

Reply:
I think the classic approach is not really a blocker, which is what an audit will cause. Just a current preference, so might as well point it out if seen.

I guess the advantage is being able to switch among generators for `make`, `ninja`, etc. with just an argument.

Not sure if there are any edge cases that will not work with (relatively newer) `cmake [--build|--install]`.

Reply:
Is it best to "Commit suggestion" or do a brand new PR?

Reply:
You can just push changes to this branch to update the PR.